api/src/openapi/schemas/project.ts

```diff
@@ -55,17 +55,23 @@ export const projectPostBody = {
       title: 'Project details',
       type: 'object',
       properties: {
-        name: {
+        project_name: {
           type: 'string'
         },
-        objectives: {
+        project_type: {
           type: 'string'
```

Comment:
type: 'number'

Reply:
yeah that's the only issue I found .. otherwise everything looks good

Reply:
Updated this to be number